core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UserStoreConfigConstants.java

```diff
@@ -178,4 +178,8 @@
     // Property for specify case insensitivity for User stores.
     public static final String CASE_INSENSITIVE_USERNAME = "CaseInsensitiveUsername";
     public static final String CASE_INSENSITIVE_USERNAME_DESCRIPTION = "Whether the username is case sensitive or not";
+
+    // Property to bypass account locking for the userstore.
+    public static final String BYPASS_ACCOUNT_LOCK = "BypassAccountLock";
+    public static final String BYPASS_ACCOUNT_LOCK_DESCRIPTION = "Enable account lock by pass for the userstore.";
```

Comment:
`Bybass`is a single word right? 
`Enable bypassing account locking for the userstore` WDYT?